README.md

```diff
@@ -36,20 +36,20 @@ This distribution includes cryptographic software. Your country may have restric
 | [util](./util) | Miscellaneous utilities. |
 
 #### Selected Binaries
-| Target | Description | Used by... |
-| :-- | :-- |:--|
-| [`fog-ingest-server`](./fog/ingest/server) | Obliviously post-processes the blockchain to organize transaction outputs according to fog hints.| 
-| [`fog-ledger-server`](./fog/ledger/server) | Obliviously serves ledger materials such as rings, merkle proofs, and key images, used to verify whether a Txo was spent, or construct new transactions. | 
-| [`fog-report-server`](./fog/report/server) | Provides the ingest enclave's Attestation Verification Report for transaction construction. | 
-| [`fog-view-server`](./fog/view/server) | Obliviously serves the post-processed Txos to clients who wish to check their balance and construct new transactions. | 
+| Target | Description |
+| :-- | :-- |
+| [`fog-ingest-server`](./fog/ingest/server) | Obliviously post-processes the blockchain to organize transaction outputs according to fog hints.|
+| [`fog-ledger-server`](./fog/ledger/server) | Obliviously serves ledger materials such as rings, merkle proofs, and key images, used to verify whether a Txo was spent, or to construct new transactions. |
+| [`fog-report-server`](./fog/report/server) | Provides the ingest enclave's Attestation Verification Report for transaction construction. |
+| [`fog-view-server`](./fog/view/server) | Obliviously serves the post-processed Txos to clients who wish to check their balance and construct new transactions. |
 
 ## Build Instructions
 
 The workspace can be built with `cargo build` and tested with `cargo test`. Either command will recognize the cargo `--release` flag to build with optimizations.
```

Comment:
Should we expand the `cargo test` instructions?  Adding a line about needing to call `service postgresql start` at the `./mob prompt` before calling `cargo test`...Does this still apply in this repo?

Reply:
Yes - excellent point!